SimPEG/EM/Analytics/NSEM.py

```diff
@@ -0,0 +1,128 @@
+import numpy as _np
```

Comment:
It was a try to make things clearer but from a user point of view.
![importana](https://user-images.githubusercontent.com/13682747/26953054-1cb11224-4c5d-11e7-8566-77a72f544727.png)
The Tab completion does not look like quite as what I assume we want to look like.

With the above 'trick', here is what I get:
![importnsem](https://user-images.githubusercontent.com/13682747/26953095-46de5962-4c5d-11e7-95c5-36153fdeea68.png)

It is easy to change back but that is might be something we want to think of.